### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by FredyArroyave12**
pokedex-v1

### CODE REVIEW

File: src/components/pokemonList/pokemonList.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        </div>
+      )}
+      <InfiniteScroll
+        dataLength={props.pokemonData.pokemons.length}
+        next={fetchPokemonList}
+        hasMore={true}
+      >
+        <ul className="pokemon__items">
+          {props.pokemonData.pokemons
+            .filter((item) => item.name.includes(props.navBar.searchContent))
```
Comment (hackeo1) [inline src/components/pokemonList/pokemonList.js:61]: filter must be done using the store

**Review by hackeo1 — CHANGES_REQUESTED** [review]
- check the naming convention on the images
- use either npm or yarn (remove one of the lock files, either package-lock.json or yarn.lock)

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (jugonz31) [inline public/index.html]: Would be good to add a custom Title  to the page

**Review by jugonz31 — APPROVED** [review]
Besides Carlos comments, it looks really good to me

**Review by hackeo1 — APPROVED** [review]

**Pull request merged**